controlplane/kubeadm/internal/workload_cluster_etcd.go

```diff
@@ -93,6 +96,10 @@ func (w *Workload) EtcdIsHealthy(ctx context.Context) (HealthCheckResult, error)
 		}
 
 		member := etcdutil.MemberForName(members, name)
+		if member == nil {
+			response[name] = errors.Wrap(err, "node connected missing from etcd members")
+			continue
+		}
```

Comment:
Was this just missed before?

Reply:
This is a scenario where etcd pod on the node is `ready` but the members list does not have that node.
This may never be the case but we are reaching `member.Alarms` below, this will avoid panic.